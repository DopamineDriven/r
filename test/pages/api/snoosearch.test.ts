import fetch from 'isomorphic-unfetch';
test('query', async () => {
	const q = 'snowboarding';
	const data = await fetch(
		`https://subreddit-search.vercel.app/api/snoosearch?q=${q}`
	);
	const res = data.json();
	expect(q).toEqual('snowboarding');
	expect(res).toBeDefined();
	expect(q).toMatchSnapshot();
	expect(res).toMatchSnapshot();
});
