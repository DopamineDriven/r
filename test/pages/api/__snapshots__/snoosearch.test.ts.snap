// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`query 1`] = `"snowboarding"`;

exports[`query 2`] = `Promise {}`;
